modules/tools/phigaro.nf

```diff
@@ -0,0 +1,28 @@
+process phigaro {
+      label 'phigaro'
+      errorStrategy 'ignore'
+    input:
+      tuple val(name), path(fasta) 
+    output:
+      tuple val(name), path("output/phigaro_*.txt")
+      tuple val(name), path("output/", type: 'dir')
+    script:
+      """
+      /root/miniconda3/bin/phigaro -f ${fasta} -o output -t ${task.cpus} --wtp --config /root/.phigaro/config.yml
```

Comment:
Do you need this /root path? you cant add phigaro to e.g. bin/ ?  or use "ENV" in the docker file? this path thing looks a bit "unstable"